themes/bootstrap3/templates/Recommend/SideFacets/checkbox-filters.phtml

```diff
@@ -1,6 +1,18 @@
 <?php foreach ($checkboxFilters as $current): ?>
-  <div class="facet" >
-    <a class="checkbox-filter<?php if(!($results->getResultTotal() > 0 || $current['selected'] || $current['alwaysVisible'])): ?> hidden<?php endif; ?>" href="<?=$current['selected'] ? $results->getUrlQuery()->removeFilter($current['filter']) : $results->getUrlQuery()->addFilter($current['filter']);?>" data-facet="<?=$this->escapeHtmlAttr($current['filter'])?>" data-facet="<?=$this->escapeHtmlAttr($current['filter'])?>">
+  <div class="facet">
```

Comment:
Are there functional changes here, or is this just a refactor for readability? If the latter, maybe we can address it separately.

Reply:
It added support for disabled checkbox facets, but it doesn't make sense anymore when there can be more than two. Reverted.